met-web/src/components/engagement/view/PreviewBanner.tsx

```diff
@@ -66,11 +66,12 @@ export const PreviewBanner = () => {
                         Preview Engagement{!isDraft && ' - Published'}
                     </Typography>
                     <ConditionalComponent condition={isDraft}>
-                        <Grid item container rowSpacing={isSmallScreen ? 2 : 0}>
+                        <Grid item container rowSpacing={isSmallScreen ? 2 : 0.5}>
```

Comment:
I think it's good to always specify the direction in the grid containers. I think if you don't it defaults to direction="row" anyway, but better to specify for clarity

Reply:
will create a seperate task for this